community-test/src/test/scala/CommunityDottySuite.scala

```diff
@@ -156,11 +156,7 @@ class CommunityDottySuite extends FunSuite {
 
   final def dottyExclusionList = List(
     // [scalameta] erased modifier - for now used internally, will be available in 3.1
-    "library/src/scala/compiletime/package.scala",
-    // if then - else without outdentation before else.
-    // it's unclear what to do in this case
-    // https://github.com/lampepfl/dotty/issues/10372
```

Comment:
I would close this issue as we actually support this syntax anyway, WDYT?

Reply:
Good idea, closed it despite it being a bit weird.